openslides/assignments/models.py

```diff
@@ -249,7 +271,14 @@ def create_poll(self):
         poll = self.polls.create(
             description=self.poll_description_default,
             pollmethod=pollmethod)
-        poll.set_options({'candidate': user} for user in candidates)
+        options = []
+        for user in candidates:
```

Comment:
Candidates is already a db evaluation so we should not evaluate the db again and again inside the loop. Try to solve this with only one query for all users / related users. Maybe use prefetch_related (https://docs.djangoproject.com/en/1.10/ref/models/querysets/#prefetch-related).